pandas/core/generic.py

```diff
@@ -5538,6 +5538,24 @@ def astype(
         0    10
         1     2
         dtype: int64
+
+        Create a series of dates:
+
+        >>> ser_date = pd.Series(pd.date_range('20200101', periods=3))
+        >>> ser_date
+        0   2020-01-01
+        1   2020-01-02
+        2   2020-01-03
+        dtype: datetime64[ns]
+
+        Convert to datetime type with time zone
```

Comment:
More generally: `Datetimes are localized to UTC first before converting to the specified timezone.`

Reply:
I think the entire commentary can just be `Datetimes are localized to UTC first before converting to the specified timezone.`

You can remove the `Convert to datetime type with time zone` and `(Behavior - ` portion